.eslintrc.json

```diff
@@ -105,6 +105,7 @@
         "object-property-newline": "off",
         "no-continue": "off",
         "no-extra-semi": "off",
-        "no-spaced-func": "off"
+        "no-spaced-func": "off",
+	"no-useless-return": "off"
```

Comment:
@julianlam made a commit specifically to change that and even edited it directly on github. No idea why it's doing that.